ios/PurchasesHybridCommon/PurchasesHybridCommonIntegrationTests/CommonFunctionality+async.swift

```diff
@@ -20,4 +19,57 @@ extension CommonFunctionality {
         }
     }
 
+    static func purchase(product productIdentifier: String,
```

Comment:
we could have these live as part of the public API, but as of right now we wouldn't really be able to use them because they're only compatible with iOS 13. So having them here makes sense 👍